IaC/lamda_module_template.tf

```diff
@@ -0,0 +1,149 @@
+resource "aws_iam_role" "LambdaRole" {
```

Comment:
This file should be moved to a folder called something like `lambda_api_module`, and then the file renamed to `resources.tf`. Module and template mean the same thing so it's repetitive.